docs/release-notes/rockcraft-1-7-0.rst

```diff
@@ -0,0 +1,147 @@
+.. _release-1.7.0:
+
+Rockcraft 1.7.0 release notes
+=============================
+
+20 December 2024
+
+Learn about the new features, changes, and fixes introduced in Rockcraft 1.7.0.
+For information about the Rockcraft release cycle, see the
+:ref:`release_policy_and_schedule`.
+
+
+Requirements and compatibility
+------------------------------
+
+To run Rockcraft, a system requires the following minimum hardware and
+installed software. These requirements apply to local hosts as well as VMs and
+container hosts.
+
+
+Minimum hardware requirements
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- AMD64, ARM64, ARMv7-M, RISC-V 64-bit, PowerPC 64-bit little-endian, or S390x
+  processor
+- 2GB RAM
+- 10GB available storage space
+- Internet access for remote software sources and the Snap Store
+
+
+Platform requirements
+~~~~~~~~~~~~~~~~~~~~~
+
+.. list-table::
+  :header-rows: 1
+  :widths: 1 3 3
+
+  * - Platform
+    - Version
+    - Software requirements
+  * - GNU/Linux
+    - Popular distributions that ship with systemd and are `compatible with
+      snapd <https://snapcraft.io/docs/installing-snapd>`_
+    - systemd
+
+
+What's new
+----------
+
+Rockcraft 1.7.0 brings the following features, integrations, and improvements.
+
+
+Support for remote builds
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We added support for remote builds, which leverage off-site compute for
+building rocks and can build for multiple architectures with a single command.
+Remote builds are hosted on Launchpad.
+
+This feature is experimental and subject to change, and more information is
+available by running ``rockcraft remote-build --help``. Feel free to provide
+feedback on the `Ubuntu Discourse`_ forum.
+
+Minor features
+--------------
+
+Rockcraft 1.7.0 brings the following minor changes.
+
+
+``init`` command
+~~~~~~~~~~~~~~~~
+
+The default base of the project file generated by ``rockcraft init`` has been
+updated from ``ubuntu@22.04`` to ``ubuntu@24.04``.
+
+
+Rockcraft snap
+~~~~~~~~~~~~~~
+
+The base of the Rockcraft snap was updated from core22 to core24.
+
+
+Go extension
+~~~~~~~~~~~~
+
+A new :ref:`tutorial <build-a-rock-for-a-go-application>` and :ref:`reference
+<go-framework-reference>` are available for the go-framework extension.
+
+
+OCI image configuration
+~~~~~~~~~~~~~~~~~~~~~~~
+
+When using ``run-user: _daemon_``, the value of the ``Config.User`` key in the
+generated rock has been updated from the user name (``_daemon_``) to its
+numeric ID.
+
+
+12-factor extensions
+~~~~~~~~~~~~~~~~~~~~
+
+We added support for Gunicorn async workers on the Django and Flask 12-factor
```

Comment:
> 12-factor

Should we call them "12-factor extensions"? "Django and Flask extensions, for supporting 12-factor apps" might be clearer.

Reply:
reworded this: kept as Django and Flask, and updated the title to `12-factor app extensions`